CRM/Civicase/Service/CaseCategoryMenu.php

```diff
@@ -16,6 +16,9 @@ class CRM_Civicase_Service_CaseCategoryMenu {
    *   Case Type category name.
    */
   public function createItems($caseTypeCategoryName) {
+    $caseTypeCategoryName = strtolower($caseTypeCategoryName);
```

Comment:
There is a little issue here. The case category name is used elsewhere in this code apart for labels but here we are making it all lowercase. The case category name might be something like `caTegory` and it will be used for menu names and all. I suggest we just do something like below and leave the original case category name variable untouched.

```php
$labelForMenu = strtolower(ucfirst($caseTypeCategoryName));
```

Reply:
Yep, this was one of the things that did not convince me. Fixed: https://github.com/compucorp/uk.co.compucorp.civicase/pull/743/commits/c3a4947f5e9d201b18c8efb2362fe7a7c77fa2b1